addons/sys_antenna/XEH_postInit.sqf

```diff
@@ -1,17 +1,18 @@
 #include "script_component.hpp"
-NO_DEDICATED;
 
-private _components = configFile >> "CfgAcreComponents";
+if (hasInterface) then {
```

Comment:
`if (!hasInterface) exitWith {};` would make more sense here, less indentation needed. It's only for preInit that we can't just exit early due to `˙ADDON = false/true;`.